web3-dev/smart-contracts/autonomous-smart-contract.rst

```diff
@@ -3,17 +3,17 @@
 Autonomous Smart Contract
 =========================
 
-In smart-contract world of Massa, you can create smart-contracts that can be called in the future.
+In smart-contract world of Massa, you can create smart-contracts that can be triggered in the future.
 
 This system works by sending messages in the future specifying the address and the function you want to trigger.
```

Comment:
You don't really "send" the message in the future.

Reply:
```suggestion
This system allows an SC to schedule future executions. Address, executed functions and execution time range must be specified.
```